src/plonemeeting/portal/core/migrations/migrate_to_1006.py

```
@@ -25,17 +25,19 @@ def _transform_delib_categories_to_dict(self):
-                    if (not include or parameter in include) and parameter not in exclude:
+                    if (
+                        not include or parameter in include
+                    ) and parameter not in exclude:
                         res.append(
                             {
-                                'parameter': parameter,
+                                "parameter": parameter,
                                 # special behavior for fullobjects parameter
                                 # that does not have a value
-                                'value': len(sub) > 1 and sub[1].strip() or "true"
+                                "value": len(sub) > 1 and sub[1].strip() or "true",
```

Ruff (0.15.14)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)